unittests.py

```diff
@@ -42,6 +42,29 @@ def test_alive_greater_than_150(self):
     def test_dead_greater_than_150(self):
         self.assertFalse(self.verifier(examples.examplePersonDeadOver150))
 
+class TestForBigamy(unittest.TestCase):
+    def testMarriedPerson(self):
+        self.assertFalse(noBigamy(examples.examplePersonMarried))
+    def testNonMarriedPerson(self):
+        self.assertTrue(noBigamy(examples.examplePersonNonMarried))
+        
+class TestForBigamy(unittest.TestCase):
+    def setUp(self):
+        self.familiesWithoutBigamy = {
+            '@F1@': examples.exampleFamilyWithWidow, #Hus ID = 4, Wife ID = 5
+            '@F2@': examples.exampleImproperFamilyDivorced, #Hus ID = 7, Wife ID = 3
+            '@F3@': examples.exampleFamilyTogether #Hus ID = 2, Wife ID = 1
+        }
+        self.familiesWithBigamy = {
+            '@F1@': examples.exampleFamilyWithWidow, #Hus ID = 4, Wife ID = 5
+            '@F2@': examples.exampleImproperFamilyDivorced, #Hus ID = 7, Wife ID = 3
+            '@F3@': examples.exampleFamilyDivorced #Hus ID = 7, Wife ID = 3
+        }
+    def testPositiveBigamy(self):
+        self.assertTrue(project03.verifyNoBigamy(self.familiesWithoutBigamy['@F1@'])

```

Comment:
This doesn't look like it will compile, a closing parenthesis is missing, did you run this?